spec/plans/results.fmf

```diff
@@ -60,6 +60,29 @@ description: |
          name: client-1
          role: clients
 
+      # Represents results of all test checks executed as driven by test's `check`
+      # key. Fields have the same meaning as fields of the "parent" test result, but
+      # relate to each check alone.
+      checks:
```

Comment:
Shall we keep singular form similarly as we do for the `log` key?

Reply:
Thanks, result schema needs adjustments as well, right?

Reply:
Right, fixed in b617e84a

Reply:
Thanks, and one more plural fixed in d0bfc062.